test/test_component.py

```diff
@@ -418,5 +463,138 @@ def test_sep055(self):
         doc.add([part_in_backbone_component_linear, part_in_backbone_seq])
         assert doc_diff(doc, hlc_doc) == 0, f'Constructor Error: Linear {identity_pib}'
 
+        hlc_doc = sbol3.Document()
```

Comment:
This enormous test routine should be broken into multiple separate routines.

Reply:
separated routines now